tests/test_loss/test_regression_losses.py

```diff
@@ -63,3 +63,6 @@ def test_wing_loss():
     assert torch.gt(
         loss(fake_pred, fake_label, torch.ones_like(fake_label)),
         torch.tensor(.5))
+
+
+test_wing_loss()
```

Comment:
This line is not needed in the final version.

BTW for better experience in debugging unittest, try vscode. 

![image](https://user-images.githubusercontent.com/9464825/108651064-f6bb1080-74fb-11eb-817a-e2f40a264421.png)